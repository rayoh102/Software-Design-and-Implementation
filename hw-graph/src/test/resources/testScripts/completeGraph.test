# Test adding multiple nodes and types of edges on a graph

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1
AddNode graph1 n2

# Add edges
AddEdge graph1 n1 n1 e1
AddEdge graph1 n1 n2 e2
AddEdge graph1 n2 n2 e1
AddEdge graph1 n2 n1 e2

# Make sure nodes exist
ListNodes graph1

# Check connections
ListChildren graph1 n1
ListChildren graph1 n2