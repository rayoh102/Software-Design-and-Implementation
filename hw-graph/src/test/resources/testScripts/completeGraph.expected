# Test adding multiple nodes and types of edges on a graph

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1
added node n2 to graph1

# Add edges
added edge e1 from n1 to n1 in graph1
added edge e2 from n1 to n2 in graph1
added edge e1 from n2 to n2 in graph1
added edge e2 from n2 to n1 in graph1

# Make sure nodes exist
graph1 contains: n1 n2

# Check connections
the children of n1 in graph1 are: n1(e1) n2(e2)
the children of n2 in graph1 are: n1(e2) n2(e1)